build/cloud/08-service.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: v1
```

Comment:
Should we delay this for now since support for K8S service isn't there in edge_core yet?

Reply:
This service definition is used to expose edge controller ( port 10000 [cloud hub]), user should modify the service type to `LoadBalancer` or use `ExternalIPs` or `NodePort` to let edge core connect to.

https://kubernetes.io/docs/concepts/services-networking/#loadbalancer
https://kubernetes.io/docs/tutorials/stateless-application/expose-external-ip-address/

Reply:
I think I finally got your point. Your mean in a k8s cluster, there is no normal node at all. Only edge node. So the `service` won't work.

In this scenario, It's not a good idea to deploy edge controller as deployment.

Reply:
for now since we don't support service , we can avoid the service definition for the cloud module. maybe in future we can add this if we add the support in edge_core.